src/main/kotlin/no/nav/familie/ks/sak/kjerne/logg/LoggService.kt

```diff
@@ -113,4 +114,55 @@ class LoggService(
             )
         )
     }
+
+    fun opprettSettPåVentLogg(behandling: Behandling, årsak: String) {
+        lagreLogg(
+            Logg(
+                behandlingId = behandling.id,
+                type = LoggType.BEHANDLIG_SATT_PÅ_VENT,
+                rolle = SikkerhetContext.hentRolletilgangFraSikkerhetscontext(
+                    rolleConfig,
+                    BehandlerRolle.SAKSBEHANDLER
+                ),
+                tekst = "Årsak: $årsak"
+            )
+        )
+    }
+
+    fun opprettOppdaterVentingLogg(behandling: Behandling, endretÅrsak: String?, endretFrist: LocalDate?) {
+        val tekst = if (endretFrist != null && endretÅrsak != null) {
+            "Frist og årsak er endret til $endretÅrsak og ${endretFrist.tilKortString()}"
+        } else if (endretFrist != null) {
+            "Frist er endret til ${endretFrist.tilKortString()}"
+        } else if (endretÅrsak != null) {
+            "Årsak er endret til $endretÅrsak"
+        } else {
+            return
+        }
+
+        lagreLogg(
+            Logg(
+                behandlingId = behandling.id,
+                type = LoggType.VENTENDE_BEHANDLING_ENDRET,
+                rolle = SikkerhetContext.hentRolletilgangFraSikkerhetscontext(
+                    rolleConfig,
+                    BehandlerRolle.SAKSBEHANDLER
+                ),
+                tekst = tekst
+            )
+        )
+    }
+
+    fun gjenopptaBehandlingLogg(behandling: Behandling) {
```

Comment:
```suggestion
    fun opprettBehandlingGjenopptattLogg(behandling: Behandling) {
```
Hmm, det opprettes logg når man kaller på denne metoden, så jeg synes at det hadde vært litt bedre å få det fram i metodenavnet